recipes/gammaray/meta.yaml

```diff
@@ -0,0 +1,73 @@
+
+{% set name = "gammaray" %}
+{% set version = "2.11.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/KDAB/GammaRay/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+    sha256: bba4f21a2bc81ec8ab50dce5218c7a375b92d64253c690490a6fcb384c2ff9f3
+
+  - url: http://ftp.vim.org/languages/qt/development_releases/prebuilt/llvmpipe/windows/opengl32sw-64-mesa_12_0_rc2.7z   # [win64]
+    sha256: 2a0d2f92c60e0962ef5f6039d3793424c6f39e49ba27ac04a5b21ca4ae012e15                                             # [win64]
+    folder: opengl32sw                                                                                                   # [win64]
+
+build:
+  number: 0
+  rpaths:
+    - lib  # [osx]
+    - GammaRay.app/Contents/Frameworks  # [osx]
+  missing_dso_whitelist:  # [osx]
+    - /usr/lib/libbsm.0.dylib  # [osx]
+    - /usr/lib/libcups.2.dylib  # [osx]
+    - /usr/lib/libresolv.9.dylib  # [osx]
+    - /usr/lib/libsandbox.1.dylib  # [osx]
+    - '/System/Library/Frameworks/CoreLocation.framework/**'  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ cdt('mesa-libgl-devel') }}      # [linux]
+    - {{ cdt('mesa-libegl-devel') }}     # [linux]
+    - {{ cdt('mesa-dri-drivers') }}      # [linux]
+    - {{ cdt('libselinux-devel') }}      # [linux]
+    - {{ cdt('libxext-devel') }}         # [linux]
+    - {{ cdt('libxdamage') }}            # [linux]
+    - {{ cdt('libxdamage-devel') }}      # [linux]
+    - {{ cdt('libxfixes') }}             # [linux]
+    - {{ cdt('libxfixes-devel') }}       # [linux]
+    - {{ cdt('libxxf86vm') }}            # [linux]
+    - {{ cdt('libxxf86vm-devel') }}      # [linux]
+  host:
+    - cmake
+    - qt
+  run:
+    - qt
+  run_constrained:                                            # [osx]
+    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.12") }}  # [osx]
+
+test:
+  commands:
+    - gammaray --version  # [win64]
+
+about:
+  home: https://github.com/KDAB/GammaRay
+  license: GPL-3.0-or-later
+  license_family: GPL
+  license_file: LICENSE.GPL.txt
```

Comment:
License and copyright file for `opengl32sw`?

Reply:
I copied the instructions over the qt-feedstock, I don't know if they are packaging it there? Otherwise, it seems to be MIT: https://doc.qt.io/qt-5/qtgui-attribution-opengl-es2-headers.html

Reply:
Do I need to include the copyright notice, or do I need to create a separate LICENSE file containing both the GPL license for gammaray and the MIT for OpenGL?

Reply:
llvmpipe's license. How i it built? which software is included there? I assume LLVM is which is Apache-2.0 or UIUC/NCSA depending on the version. Which LLVM version is used?

Reply:
According to https://docs.mesa3d.org/license.html, it would seem that llvmpipe is MIT, since it is a driver of Mesa